frontend/server/src/main/java/org/pytorch/serve/util/ConfigManager.java

```diff
@@ -362,8 +362,8 @@ public int getJobQueueSize() {
         return getIntProperty(TS_JOB_QUEUE_SIZE, 100);
     }
 
-    public int getNumberOfPriorities() {
-        return getIntProperty(TS_NUMBER_OF_PRIORITIES, 1);
+    public float getHighPrioProb() {
+        return getFloatProperty(TS_HIGH_PRIORITY_PROBABILITY, 0.67f);
```

Comment:
Should we do some validation here, or some clamping if the value is out of range? Maybe we should ensure that this value will end up in the range `[0.0f, 1.0f]` or `[0.5f, 1.0f]`

Reply:
I can add some checks to raise an error if the value is invalid.